swift-extensions/UISwitchExtensions.swift

```diff
@@ -15,23 +15,16 @@ extension UISwitch {
             viewModel = value
             guard let toggleSwitchViewModel = value else { return }
 
-            self.addTarget(self, action: #selector(UISwitch.onValueChanged), for: .valueChanged)
             observe(toggleSwitchViewModel.isOn) { [weak self] (value: Bool) in
-                self?.isOn = value
+                if(self?.isOn != value) {
```

Comment:
```suggestion
                if (self?.isOn != value) {
```
Make sure to run swiftlint to catch these formatting errors.